model/task/task.go

```diff
@@ -1264,6 +1252,12 @@ func SetTasksScheduledTime(tasks []Task, scheduledTime time.Time) error {
 	return nil
 }
 
+// GetTaskIdBetweenIds retrieves the task id between two tasks that are of the same.
+func GetTaskIdBetweenTasks(t1, t2 Task) (*Task, error) {
+	mid := (t1.RevisionOrderNumber + t2.RevisionOrderNumber) / 2
+	return FindOne(db.Query(ByRevisionOrderNumber(t1.BuildVariant, t2.DisplayName, t1.Project, t1.Requester, mid)))
```

Comment:
Is it significant that it's using `t2`'s display name but `t1`'s fields for everything else?

Reply:
Not at all, that is a typo 😆